ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_pooling_functions.py

```diff
@@ -256,6 +247,7 @@ def test_torch_max_pool1d(
         max_side=4,
         explicit_or_str_padding=True,
         return_dilation=True,
+        data_format="channel_first",
     ).filter(lambda x: x[4] != "VALID" and x[4] != "SAME"),
```

Comment:
Again I think if we set `only_explicit_padding=True` we can avoid this filtering.